python/paddle/fluid/tests/unittests/test_reverse_op.py

```diff
@@ -99,241 +91,6 @@ def initTestCase(self):
-    def setUp(self):
-        self.shapes = [[5, 25], [5, 20], [5, 5]]
-        self.place = (
-            fluid.CUDAPlace(0)
-            if fluid.is_compiled_with_cuda()
-            else fluid.CPUPlace()
-        )
-        self.exe = fluid.Executor(self.place)
-
-    def run_program(self, arr_len, axis=0):
-        main_program = fluid.Program()
-
-        with fluid.program_guard(main_program):
-            inputs, inputs_data = [], []
-            for i in range(arr_len):
-                x = fluid.data("x%s" % i, self.shapes[i], dtype='float32')
-                x.stop_gradient = False
-                inputs.append(x)
-                inputs_data.append(
-                    np.random.random(self.shapes[i]).astype('float32')
-                )
-
-            tensor_array = fluid.layers.create_array(dtype='float32')
-            for i in range(arr_len):
-                idx = fluid.layers.array_length(tensor_array)
-                fluid.layers.array_write(inputs[i], idx, tensor_array)
-
-            reverse_array = fluid.layers.reverse(tensor_array, axis=axis)
-            output, _ = fluid.layers.tensor_array_to_tensor(reverse_array)
-            loss = paddle.sum(output)
-            fluid.backward.append_backward(loss)
-            input_grads = list(
-                map(
-                    main_program.global_block().var,
-                    [x.name + "@GRAD" for x in inputs],
-                )
-            )
-
-            feed_dict = dict(zip([x.name for x in inputs], inputs_data))
-            res = self.exe.run(
-                main_program,
-                feed=feed_dict,
-                fetch_list=input_grads + [output.name],
-            )
-
-            return np.hstack(inputs_data[::-1]), res
-
-    def test_case1(self):
-        gt, res = self.run_program(arr_len=3)
-        self.check_output(gt, res)
-        # test with tuple type of axis
-        gt, res = self.run_program(arr_len=3, axis=(0,))
-        self.check_output(gt, res)
-
-    def test_case2(self):
-        gt, res = self.run_program(arr_len=1)
-        self.check_output(gt, res)
-        # test with list type of axis
-        gt, res = self.run_program(arr_len=1, axis=[0])
-        self.check_output(gt, res)
-
-    def check_output(self, gt, res):
-        arr_len = len(res) - 1
-        reversed_array = res[-1]
-        # check output
-        np.testing.assert_array_equal(gt, reversed_array)
-        # check grad
-        for i in range(arr_len):
-            np.testing.assert_array_equal(res[i], np.ones_like(res[i]))
-
-    def test_raise_error(self):
-        # The len(axis) should be 1 is input(X) is LoDTensorArray
-        with self.assertRaises(Exception):
-            self.run_program(arr_len=3, axis=[0, 1])
-        # The value of axis should be 0 is input(X) is LoDTensorArray
-        with self.assertRaises(Exception):
-            self.run_program(arr_len=3, axis=1)
-
-
-class TestReverseAxisTensor(UnittestBase):
```

Comment:
shall we keep `TestReverseAxisTensor` and replace `paddle.fluid.layers.reverse` with `paddle.reverse` instead of delete the whole class? the same as `TestReverseAxisListTensor`, `TestReverseDoubleGradCheck` and `TestReverseTripleGradCheck`

Reply:
> shall we keep `TestReverseAxisTensor` and replace `paddle.fluid.layers.reverse` with `paddle.reverse` instead of delete the whole class? the same as `TestReverseAxisListTensor`, `TestReverseDoubleGradCheck` and `TestReverseTripleGradCheck`

Resolve

Reply:
paddle.reverse(flip)的单测已经覆盖这些case, 顾这些单测在此处不再需要